pyccel/parser/semantic.py

```diff
@@ -2389,6 +2396,7 @@ def _visit_FunctionCall(self, expr):
             # TODO treat parametrized arguments.
 
             expr = ConstructorCall(method, args, cls_variable)
+            self._allocs.append([expr])
```

Comment:
Why not append `cls_variable` directly? Why does this need to be a list?

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1508/commits/4464522edd12b45c32f6d7ee46ababb878edcd2f.